content/en/docs/concepts/scheduling-eviction/topology-spread-constraints.md

```diff
@@ -96,14 +96,6 @@ your cluster. Those fields are:
   A domain is a particular instance of a topology. An eligible domain is a domain whose
   nodes match the node selector.
 
-  {{< note >}}
-  The `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
-  enables `minDomains` for pod topology spread. Starting from v1.28,
-  the `MinDomainsInPodTopologySpread` gate 
-  is enabled by default. In older Kubernetes clusters it might be explicitly
-  disabled or the field might not be available.
-  {{< /note >}}
-
```

Comment:
Try this:
```markdown
  <!-- OK to remove this note once v1.29 Kubernetes is out of support -->
  {{< note >}}
  Before Kubernetes v1.30, the `minDomains` field was only available if the
  `MinDomainsInPodTopologySpread` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
  was enabled (default since v1.28). In older Kubernetes clusters it might be explicitly
  disabled or the field might not be available.
  {{< /note >}}
```

Reply:
Done.